services/mailer.js

```diff
@@ -1,13 +1,18 @@
 const mailgun = require("mailgun-js");
 const logger = require("../lib/log")(__filename);
-const getEnvVar = require("../lib/getEnvVar");
```

Comment:
I can't remember why I did this..
I thought the reason was because we had to hijack the env variable when we test or something. So I reviewed the code again but it doesn't seem that way. So weird!

Reply:
I thought it had something to do with tests too after I tried to add new ones to mailer. With separate getEnv you could check env variables separately and without it I need to pass them as arguments to email.sendPasswordResetEmail.